_schemaV2.graphql

```diff
@@ -10542,6 +10546,13 @@ type FormattedDaySchedules {
 # a formatted String. It does not try to coerce the type.
 scalar FormattedNumber
 
+# The collector's interest in the gallery program
+type GalleryInterest {
+  alertsCount: Int!
+  followedArtistsCount: Int!
+  savedArtworksCount: Int!
+}
```

Comment:
**question (non-blocking):** Curious if we can adopt here the "wide" Metaphysics counts type?
```
galleryInterest {
    counts {
        name 
        count 
    }
} 
```

Reply:
not only these counts I think also the query `inquiryCount` would be nice if could be nested under `counts`

Reply:
Right you definitely don't want to incur the performance penalty of invoking the resolvers for every count when you want one. I do like the pattern of nesting counts within a counts type though. We do this in a few other places.